packages/sdk/package.json

```diff
@@ -37,13 +37,15 @@
     }
   ],
   "dependencies": {
+    "@types/highland": "^2.12.13",
```

Comment:
nit: this should be a dev dependency since the types are only needed at compile time.